worker/src/Worker.cpp

```diff
@@ -20,24 +20,25 @@ Worker::Worker(::Channel::ChannelSocket* channel, PayloadChannel::PayloadChannel
 	this->channel->SetListener(this);
 
 	// Set us as PayloadChannel's listener.
-	this->payloadChannel->SetListener(this);
+	//this->payloadChannel->SetListener(this);
```

Comment:
Here and below in Worker.cpp please just remove code instead of commenting it out for better readability. Unless this is smth temporary i.e. you plan to uncomment it back later, or work on it.

Reply:
Need to revisit the requirements to implement the payload channel before I remove it. Will add a to-do comment.